Sources/Modules/EternalResolve/Items/Weapons/StabbingSwords/Projectiles/StabbingProjectile.cs

```diff
@@ -97,7 +97,7 @@ public override void AI()
             Projectile.soundDelay--;
             if (Projectile.soundDelay <= 0)
             {
-                SoundEngine.PlaySound(SoundID.Item1, Projectile.Center);
+                SoundEngine.PlaySound(SoundID.Item1.WithVolumeScale(0.5f).WithPitchOffset(Main.rand.NextFloat(-0.6f, 0.6f)), Projectile.Center);
```

Comment:
I like it, but the volume no longer matches Starlight, an already existing bayonet dropped from the Empress of Light. 
In a different code file, you can try to override the sound for Starlight.
Do you want me to make the change for you?

Reply:
yes of course.It's nice that you can change starlight bayonet.

Reply:
> yes of course.It's nice that you can change starlight bayonet.

So making the change was harder than I thought, because I thought that it was a global item change I need to make. I found that it was a global projectile thing, and I had no idea how to make the sound adjustment work the way it should (it's my first time working with global projectiles)

![image](https://github.com/Solaestas/Everglow/assets/70764728/4af06eb5-c723-4953-9d18-196856dd8937)

I can instead make it so the volume scale and pitch offset for the Stabbing Projectiles are no longer applied in order to match the Starlight weapon (which is classified as a rapier, but we called it a bayonet). The sound from the bayonets are too quiet for me anyways with both music and sound at the same volume.